### PR TITLE
MIM-166: adds notifications from Mimer.nu on cards in Kanban

### PR DESCRIPTION
**Pull request by johanneskarlsson**

### CODE REVIEW

**Review by henrikhenrikpersson — APPROVED** [review]

File: onecore_maintenance_extension/models/maintenance.py [hunk 1]
```diff
@@ -90,6 +89,26 @@ class OneCoreMaintenanceRequest(models.Model):
...
+    def _compute_new_mimer_notification(self):
+        for record in self:
+            message_ids = record.message_ids.ids
+
+            unread_mimer_notifications = self.env['mail.notification'].search([
+                ('mail_message_id', 'in', message_ids),
+                ('res_partner_id', '=', self.env.user.partner_id.id),
+                ('is_read', '=', False),
+                ('notification_type', '=', 'inbox'),
+                ('mail_message_id.author_id.user_ids.login', '=', 'odoo@mimer.nu')
```
Comment (lalmqvist) [inline onecore_maintenance_extension/models/maintenance.py:108]: we should make sure that we have this user in all env:s. Can we add it via some hook?

Comment (lalmqvist) [reply onecore_maintenance_extension/models/maintenance.py:108]: New story maybe?

File: onecore_maintenance_extension/views/maintenance_views.xml [hunk 2]
```diff
@@ -226,11 +226,26 @@
...
                     <field name="master_key" />
+                    <field name="new_mimer_notification" />
                 </xpath>
                 <!-- Remove existing fields regarding equipment -->
                 <xpath expr="//div[@class='o_kanban_record_body']/span[1]" position="replace" />
                 <xpath expr="//div[@class='o_kanban_record_body']/span[2]" position="replace" />
                 <xpath expr="//div[@class='o_kanban_record_bottom']" position="replace" />
+                <xpath expr="//div[@class='o_kanban_record_top']" position="replace">
+                  <div class="o_kanban_record_top">
+                    <div class="w-100 d-flex justify-content-between align-items-center">
```
Comment (lalmqvist) [inline onecore_maintenance_extension/views/maintenance_views.xml:237]: Use the class `.mimer-badge` in `onecore_maintenance_extension/static/src/scss/mimer_styles.scss`

**Review by lalmqvist — CHANGES_REQUESTED** [review]
See comments!

File: onecore_maintenance_extension/models/maintenance.py (hunk 1 above)
Comment (johanneskarlsson) [reply onecore_maintenance_extension/models/maintenance.py:108]: Yes!

File: onecore_maintenance_extension/views/maintenance_views.xml (hunk 2 above)
Comment (johanneskarlsson) [reply onecore_maintenance_extension/views/maintenance_views.xml:237]: 🤯

**Pull request merged**